projects/ngneat/helipopper/src/lib/tippy.directive.ts

```diff
@@ -185,6 +185,12 @@ export class TippyDirective implements OnChanges, AfterViewInit, OnDestroy, OnIn
     return `${this.host.nativeElement.getBoundingClientRect().width}px`;
   }
 
+  private createInstanceOnNonNilValue() {
+    if (this.globalConfig.disableOnNilValue && this.content !== null && this.content !== undefined) {
```

Comment:
Why do we need a config for it? It doesn't make sense anyway to show a tooltip when the value is nil. We can make it the default functionality.

Reply:
> It doesn't make sense anyway to show a tooltip when the value is nil. We can make it the default functionality.

We absolutely can. I tried to keep the existing behaviour as is and add the new behaviour via global config. I agree that it doesn't make sense at all. That's why I tried to discuss the idea first. So if you agree, I will remove the global config part and stick to skipping the tooltip creation, if the content is nil.

Reply:
Sure

Reply:
I have removed the global config and reworked the docs accordingly.